### ISSUE TITLE
Ability to change speedometer, music play list

### ISSUE THREAD

**Issue by haywire-zz** [zenyagami/car-dashdroid#7]
Allow the option for the speedometer to be used as a compass that displays N NE S SE SW SE E W as you drive

**Comment by zenyagami** [comment]
put the compass on the speedometer space?

**Comment by haywire-zz** [comment]
Yes in the options you have to allow speedometer, or music player, it would be nice if you could add the option to add a compass option
